gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyLogging.java

```diff
@@ -94,4 +104,10 @@ private String getVoltageCheckResult(DichotomyDirection direction, Optional<Volt
         }
         return NONE;
     }
+
+    private String getTimestampLocalized(OffsetDateTime timestamp) {
+        OffsetDateTime localTime = java.time.OffsetDateTime.ofInstant(timestamp.toInstant(), zoneId);
```

Comment:
Your OffsetDateTime does not contain only time, so the name `localTime` seems a bit restrictive. Try with something like `localTimestamp` for example?

Reply:
what about localDateTime ?